jobs/tools/ICON_to_point.py

```diff
@@ -0,0 +1,362 @@
+import numpy as np
+import xarray as xr
+from sklearn.neighbors import BallTree
```

Comment:
`sklearn.neighbors` is not part of the conda environment. It should be added to `env/environment.yml`.

Reply:
Done. Thanks!